pkg/dbal/DbalSubscriptionConsumer.php

```diff
@@ -43,8 +78,10 @@ public function consume(int $timeout = 0): void
             throw new \LogicException('No subscribers');
         }
 
-        $timeout = (int) ceil($timeout / 1000);
-        $endAt = time() + $timeout;
+        $now = time();
+        $timeout /= 1000;
+        $deliveryId = (string) Uuid::uuid1();
```

Comment:
it should be generated per message, hence the var init should be done before fetchMessage call.